data/sources/01_areas-extended.yaml

```diff
@@ -733,9 +847,17 @@ frm-2:
   coords: { lat: 48.26826, lon: 11.67866 }
   osm: ["way/159713511"]
 
+5201:
+  coords: { lat: 48.26582, lon: 11.67346 }
+  osm: ["way/774866203"]  # OSM says: 5258
```

Comment:
If we are confident this is the right object, we could provide a patch to OSM, how sure are you, that this match is correct?

Reply:
It comes from the fact that the building name is `UYL` in OSM, which is also the building name from the NAT roomfinder and correct according to [this source](https://www.frm2.tum.de/fileadmin/w00bnv/www/content_uploads/experimental-facilities_online.pdf) (page 118 or 116, "Access to the MLZ"). So it depends on how sure we are about that the ID in the NAT Roomfinder is correct (and not outdated)